pkg/odo/cli/preference/unset.go

```diff
@@ -92,9 +92,12 @@ func NewCmdUnset(name, fullName string) *cobra.Command {
 		Short: "Unset a value in odo preference file",
 		Long:  fmt.Sprintf(unsetLongDesc, preference.FormatSupportedParameters()),
 		Example: func(exampleString, fullName string) string {
-			for _, property := range preference.GetSupportedParameters() {
-				exampleString += fmt.Sprintf("\n  %s %s", fullName, property)
-			}
+			// For now, the --help page is too large listing all unset values too.
+			// Just show one example of how to unset a value.
+			//for _, property := range preference.GetSupportedParameters() {
```

Comment:
I'm not sure it is necessary to keep old code here. You could remove the code, and explain why we only display for 1st  parameter.